panther_lights_controller/README.md

```diff
@@ -14,37 +14,173 @@ Package used to control the Husarion Panther LED panels.
 
 #### Parameters
 
-- `~animations` [*list*]: it is a required ROS parameter containing a list of defined animations.
-- `~global_brightness` [*float*, default: **1.0**]: LED global brightness.
-- `~num_led` [*int*, default: **46**]: number of LEDs in single panel.
+- `~animations` [*list*, default: None]: it is a required ROS parameter containing a list of defined animations.
 - `~controller_frequency` [*float*, default: **100**]: frequency at which the lights controller node will process animations.
-- `~test` [*bool*, default: **false**]: allows testing mode with some extra functionalities.
+- `~global_brightness` [*float*, default: **1.0**]: LED global brightness. Range between [0,1].
+- `~num_led` [*int*, default: **46**]: number of LEDs in single panel.
+- `~test` [*bool*, default: **false**]: enables testing mode with some extra functionalities.
+- `~user_animations` [*list*, default: None]: it is an optional ROS parameter containing a list of animations defined by the user.
```

Comment:
```suggestion
- `~user_animations` [*list*, default: **None**]: optional list of animations defined by the user.
```
I'm a bit lost right now how are we supposed to define it outside of YAML. But it is ok nevertheless